static/app/components/searchQueryBuilder/tokens/filter/filterOperator.tsx

```diff
@@ -85,21 +86,28 @@ function getTermOperatorFromToken(token: TokenResult<Token.FILTER>) {
 }
 
 function FilterKeyOperatorLabel({
+  keyValue,
   keyLabel,
   opLabel,
   includeKeyLabel,
 }: {
+  keyLabel: string;
+  keyValue: string;
   includeKeyLabel?: boolean;
-  keyLabel?: string;
   opLabel?: string;
 }) {
+  const {getFieldDefinition} = useSearchQueryBuilder();
+  const fieldDefinition = getFieldDefinition(keyValue);
+
   if (!includeKeyLabel) {
     return <OpLabel>{opLabel}</OpLabel>;
   }
 
   return (
     <KeyOpLabelWrapper>
-      <span>{keyLabel}</span>
+      <Tooltip title={fieldDefinition?.desc} skipWrapper>
```

Comment:
Is it possible to wrap this tooltip around the whole thing? For `is not` it looks like it only gets wrapped around `is`. Kind of an edge case though

Reply:
thanks!

Reply:
![image](https://github.com/user-attachments/assets/291f12f7-3555-4f39-b1c5-c372975a66dd)